flink-scala-shell/start-script/start-scala-shell.sh

```diff
@@ -82,13 +82,25 @@ do
     fi
 done
 
+if [ "$FLINK_IDENT_STRING" = "" ]; then
+        FLINK_IDENT_STRING="$USER"
+fi
+
 log_setting=""
 
-if [[ $1 = "yarn" ]]
+if [[ $1 = "local" ]]
+then
+    log=$FLINK_LOG_DIR/flink-$FLINK_IDENT_STRING-scala-shell-local-$HOSTNAME.log
+    log_setting="-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4j.properties -Dlogback.configurationFile=file:"$FLINK_CONF_DIR"/logback.xml"
+elif [[ $1 = "remote" ]]
+then
+    log=$FLINK_LOG_DIR/flink-$FLINK_IDENT_STRING-scala-shell-remote-$HOSTNAME.log
```

Comment:
Could we unify setting the log parameters? They differ very slightly.
How about we have a common definition:

```
log=$FLINK_LOG_DIR/flink-$FLINK_IDENT_STRING-scala-shell-$MODE-$HOSTNAME.log
log_setting="-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/$LOG4J_CONFIG -Dlogback.configurationFile=file:"$FLINK_CONF_DIR"/$LOGBACK_CONFIG"
```

Reply:
Thanks @dawidwys for review, I have fixed it